scripts/travis/Jenkinsfile

```diff
@@ -9,7 +16,7 @@ def buildClosures(arg) {
             def(dir,testEnv) = testEnv_str.split(":")
             stage("$testEnv") {
                 travisImage.inside("--privileged --cap-add=ALL") {
```

Comment:
What do these flags to?

Reply:
These flags ensure that docker has permissions and capabilities to properly run the travis tests, the tests fail without these flags.

Reply:
I know what they do, that was a prompt for a comment describing what they did ;)